system_test/common/aest_channels_SUITE_data/aeternity.yaml.mustache

```diff
@@ -32,7 +32,7 @@ chain:
     persist: true
 
 mining:
-    autostart: true
+    autostart: {{mining.autostart}}
```

Comment:
Actual fix. Without this, the settings for only `node1` to be mining do not kick in, hence - both `node1` and `node2` are mining simultaneously and are competing for leadership. This plays really bad with fast mining setup

Reply:
👍